### DISCUSSION TITLE
use NoteHighlights on MacOS

### DISCUSSION THREAD

**Discussion by jackyromano** [elvirbrk/NoteHighlight2016#164] [Q&A]
Hi, 
This tool looks great ! Is there a way to use it on macos?

Thanks in advance,
Jacky

**Comment by Neokronic** [comment]
Hi, i have the same question. Is there a way to get this utility to run on MacOS One Note? That would be a life saver!

**Comment by muzfr7** [comment]
The same request here 😀, I noticed OneNote on Windows has many features as compared to MacOs version that might be the limiting factor!!
